app/src/main/java/protect/card_locker/ImportURIHelper.java

```diff
@@ -27,6 +27,8 @@ public class ImportURIHelper {
     private static final String BARCODE_TYPE = DBHelper.LoyaltyCardDbIds.BARCODE_TYPE;
     private static final String HEADER_COLOR = DBHelper.LoyaltyCardDbIds.HEADER_COLOR;
 
+    private static final String USAGE_NUMBER = DBHelper.LoyaltyCardDbIds.USAGE_NUMBER;
+
```

Comment:
There is no need to add this to ImportURIHelper. I don't think that someone else using a card a lot should affect how often it is considered used in your installation.